0140    html-block-and-inline/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Block and Inline</title>
+</head>
+
+<body>
+  <h1>Blue America Snuggie&reg; Blanket</h1>
+  <h2>by SnuggieStore</h2>
+  <p>Price: <s>$19.99</s> $14.99</p>
+  <p>Part of out new America collection, this <em>beautiful</em> blue is already a customer favorite!</p>
+  <p>Enjoy comple <b>Freedone of Movement!</b></p>
```

Comment:
Instead of `<b>` it is more semantically correct to use `<strong>` tags to bold.